app/code/community/Yotpo/Yotpo/controllers/Review/ExportController.php

```diff
@@ -1,15 +1,20 @@
 <?php
-
 class Yotpo_Yotpo_Review_ExportController extends Mage_Adminhtml_Controller_Action
 {
     /**
      * Exports reviews by id in post param "reviews" to csv and download file when finished.
      */
     public function csvexportAction()
     {
-    	$reviews = $this->getRequest()->getPost('reviews', array());
-		$file = Mage::getModel('Yotpo_Yotpo_Model_Export_Csv')->exportReviews($reviews);
-        $this->_prepareDownloadResponse($file, file_get_contents(Mage::getBaseDir('export').'/'.$file));
+        $reviews = $this->getRequest()->getPost('reviews', array());
+        $file = Mage::getModel('Yotpo_Yotpo_Model_Export_Csv')->exportReviews($reviews);

```

Comment:
what's the difference between exportReviews & exportData?
why do we need both of them?

Reply:
Hi,
exportReviews method is getting all reviews and writing them on a file and is returning filename while exportData get all the reviews and return the array of all reviews.
Regards,
Ijaz Ali